pkg/controller/controller_test.go

```diff
@@ -213,6 +213,8 @@ func TestStripPrefixedCSIParams(t *testing.T) {
 				prefixedNodePublishSecretNamespaceKey:       "csiBar",
 				prefixedControllerExpandSecretNameKey:       "csiBar",
 				prefixedControllerExpandSecretNamespaceKey:  "csiBar",
+				prefixedDefaultSecretNameKey:                "csiBar",
+				prefixedDefaultSecretNamespaceKey:           "csiBar",
```

Comment:
Does this change cover the new logic, or just fix an existing test? I think we should add a case that gets the default value.

Reply:
This was just fix of existing test. I've added a few new tests for default secret.